citysync/app/api/views.py

```diff
@@ -17,14 +17,11 @@ def jsonsift(obj, attrlist):
 
 @api_blueprint.route("/api/orgs/all", methods=["GET"])
 def get_orgs():
-    all_orgs = Nonprofit.query.all()
-    # paginate logic
-    records = []
-    for x, org in enumerate(all_orgs):
-        records.append(org)
-        if x == 10:
-            break
-
+    page = int(request.args.get('page', None))
```

Comment:
Adding a query parameter for page was easier than expected! Nice! Keep that in mind for the other comment below